### PR TITLE
add note to readme about the limitation of ttl

### PR DESCRIPTION
**Pull request by paulovieira**

### CODE REVIEW

**Comment by Marsup** [comment]
You misunderstood me, big-time needs to be added to the library, you don't have the possibility to override it externally.

**Comment by paulovieira** [comment]
Yesh, I got it. The user would have to patch the module directly (easy fix). The note should make it clear. But let's wait to see if @cjihrig wants to add it directly or via some kind of option.

**Comment by cjihrig** [comment]
I'm open to using `big-time`. I actually tried this before in #29, but @hueniverse said no because it required babel (rest params). However, this is no longer the case in Node 6.

@hueniverse are you still 👎 on the idea?

**Comment by cjihrig** [comment]
FYI - `big-time` is now updated to drop Babel as of https://github.com/continuationlabs/big-time/pull/15

File: README.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ Lead Maintainer - [Colin Ihrig](https://github.com/cjihrig)
 
 Current version: [![Current Version](https://img.shields.io/npm/v/catbox-memory.svg)](https://www.npmjs.org/package/catbox-memory) [![Build Status](https://api.travis-ci.org/hapijs/catbox-memory.svg)](https://travis-ci.org/hapijs/catbox-memory)
 
+
```
Comment (cjihrig) [inline README.md]: Please remove these whitespace changes.

File: README.md [hunk 2]
```diff
@@ -18,3 +20,5 @@ Current version: [![Current Version](https://img.shields.io/npm/v/catbox-memory.
   `Buffer` data can be stored alongside the stringified data. `Buffer`s are not
   stringified, and are copied before storage to prevent the value from changing while
   in the cache. Defaults to `false`.
+
+Note: the maximum allowed value for `ttl` is `2^31-1`, which is around 24.8 days. If you need more than that use [big-time](https://www.npmjs.com/package/big-time).
```
Comment (cjihrig) [inline README.md]: Please remove the second sentence.

**Comment by paulovieira** [comment]
The pr has been updated.

**Pull request merged**